app/helpers/standup_meetings_helper.rb

```diff
@@ -0,0 +1,11 @@
+module StandupMeetingsHelper
+  def yesterday(date)
+    return date - 1.day if date.is_a? Date
+    Date.parse(date) - 1.day
+  end
+
+  def tomorrow(date)
+    return date + 1.day if date.is_a? Date
+    Date.parse(date) + 1.day
+  end
+end
```

Comment:
I feel like a helper isn't what you want here. Also pretty sure active support has like `date.yesterday` or `date.prev_day`. If so, you should use that. 

Gonna think where this can go feels like just a lib though.

Reply:
yeah -- I just wanted to be careful b/c sometimes it was a date and other times it was a string. I think I handled that in the controller, but...better safe than sorry. Happy to revisit.